apps/gk_species_radiation.c

```diff
@@ -166,6 +175,17 @@ void
 gk_species_radiation_moms(gkyl_gyrokinetic_app *app, const struct gk_species *species,
   struct gk_rad_drag *rad, const struct gkyl_array *fin[], const struct gkyl_array *fin_neut[])
 {
+  // compute needed moments
+  gk_species_moment_calc(&rad->lab_moms, species->local, app->local, species->f);
+  if (app->use_gpu) {
+    gkyl_mom_calc_bcorr_advance_cu(rad->bcorr_calc, &species->local, &app->local, species->f, rad->boundary_corrections);
+    gkyl_prim_lbo_calc_advance_cu(rad->coll_pcalc, &app->local, rad->lab_moms.marr, rad->boundary_corrections, rad->prim_moms);
+  } else {
```

Comment:
Move this else onto the next line for style consistency (we still have work to do changing this everywhere but the accepted style is else and else if on a new line after the brace)

Reply:
There was an additional `else` that needed to be moved for the same reason that I moved.